core/templates/dev/head/pages/about-page/about-page.module.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Module for the about page.
+ */
+
+import 'core-js/es7/reflect';
+import 'zone.js';
+
+import { Component, NgModule, StaticProvider } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { HttpClientModule } from '@angular/common/http';
+
+// This component is needed to force-bootstrap Angular at the beginning of the
+// app.
+@Component({
+  selector: 'service-bootstrap',
+  template: ''
+})
+export class ServiceBootstrapComponent {}
+
+@NgModule({
+  imports: [
+    BrowserModule,
+    HttpClientModule
+  ],
+  declarations: [
+    ServiceBootstrapComponent
+  ],
+  entryComponents: [
+    ServiceBootstrapComponent
+  ],
+})
+class AboutPageModule {
+  // Empty placeholder method to satisfy the `Compiler`.
+  ngDoBootstrap() {}
+}
+
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { downgradeModule } from '@angular/upgrade/static';
+
+const bootstrapFn = (extraProviders: StaticProvider[]) => {
+  const platformRef = platformBrowserDynamic(extraProviders);
+  return platformRef.bootstrapModule(AboutPageModule);
+};
+const downgradedModule = downgradeModule(bootstrapFn);
+
+declare var angular: any;
+
+angular.module('oppia', [
```

Comment:
Should this become an exported class (or at least a class)?

If not, when do the modules themselves get converted to Angular?

Reply:
I don't think that this is to be converted to a class (the guide also doesn't have that for the AJS module.)
The modules are upgraded in the last during the cleanup phase when we remove all the downgrade statements and the AJS module is no more needed.

Reply:
Hm, so perhaps we should include this in that step of your milestones?